code/modules/clothing/suits/labcoat.dm

```diff
@@ -56,3 +56,12 @@
 	desc = "A suit that protects against minor chemical spills. Has a black stripe on the shoulder."
 	icon_state = "labcoat_mort_open"
 	item_state = "labcoat_mort_open"
+
+/obj/item/clothing/suit/storage/labcoat/emt
+	name = "EMT labcoat"
+	desc = "A comfortable suit for paramedics. Has dark colours."
+	icon_state = "labcoat_emt_open"
+	item_state = "labcoat_emt_open"
+	species_fit = null
+	sprite_sheets = null

```

Comment:
Can go ahead and remove this and the line above once the vox sprites get added, it includes a fitted EMT labcoat